jclklib/proxy/connect_ptp4l.cpp

```diff
@@ -115,7 +115,7 @@ void event_handle()
             portState = pd->portState;
 
             /* Reset TIME_STATUS_NP data if port_state <= PASSIVE */
-            if (portState <= PASSIVE) {
+            if (portState <= TIME_RECEIVER) {
```

Comment:
need to align. slave or time_receiver . semua mesti sama.

Reply:
Agree.